### ISSUE TITLE
How to obs to OME via webrtc

### ISSUE THREAD

**Issue by alnux** [AirenSoft/OvenMediaEngine#479]
Hi i saw that OME can ingest webrtc, can you help me how can stream to OME with webrtc instead RTMP on OBS.
Thanks

**Comment by llspalex** [comment]
Standard OBS does not currently support WebRTC output, but it is in the works: https://github.com/ddrboxman/rfcs/blob/patch-1/text/0043-webrtc-output.md 

There is a [fork of OBS](https://github.com/CoSMoSoftware/OBS-studio-webrtc) that does support WebRTC output, but it uses the WHIP protocol to achieve this, which OME does not currently support - but Airensoft is aware of WHIP and has indicated that they intend to add support for it to OME: https://github.com/AirenSoft/OvenMediaEngine/issues/383#issuecomment-848362877

In short, there is currently no way to stream using WebRTC from OBS to OME. [FTL could be a good temporary solution](https://github.com/AirenSoft/OvenMediaEngine/issues/478), but probably not worth the effort.

**Comment by mpisat** [comment]
> Hi i saw that OME can ingest webrtc, can you help me how can stream to OME with webrtc instead RTMP on OBS.
> Thanks

Do you mean broadcast with WebRTC on Browser to OME?
https://demo.ovenplayer.com/demo_input.html

It's pretty easy. Documentation is very clear on WebRTC input (beta)

**Comment by alnux** [comment]
Am not a developer on that, soooo i just have to wait,  really thanks

**Comment by SangwonOh** [comment]
@alnux Hi. We will open source OvenLiveKit for Web "HTML5 Streaming SDK for OvenMediaEngine" tomorrow(19, 08 2021). Please wait a little. Thanks!

>OvenLiveKit for Web can send media streams to OvenMediaEngine. This means it can capture the user's webcam from any web browser that supports WebRTC and tranmit it directly to OvenMediaEngine's WebRTC Provider. This makes it easy to build web conferencing calls, video chats, and more.

**Comment by getroot** [comment]
This issue has been closed since it has been inactive for quite some time. If you want to continue discussing this issue, please feel free to reopen it.
